routes/admin/migration.js

```diff
@@ -0,0 +1,121 @@
+const Boom = require('boom');
+const fetch = require('node-fetch');
+const getDb = require('../../db.js').getDb;
+const server = require('../../server').getServer();
+
+const db = getDb();
+const statusUpdated = 'updated';
+const statusNotUpdated = 'not updated';
+const statusFailed = 'failed';
+
+
+module.exports = {
+  path: '/admin/migration/{tool}/{id?}',
+  method: 'GET',
+  config: {      
+    auth: 'q-auth',
+    cors: {
+      credentials: true
+    },
+    description: 'Executes migration of items in database for specified tool or for a single item with the specified id respectively',
+    tags: ['api']
+  },
+  handler: (request, reply) => {
+    const tool = request.params.tool;
```

Comment:
please test here if the tool is configured and has a baseUrl configured, return with NotImplemented otherwise.